src/Native/CMakeLists.txt

```diff
@@ -183,10 +183,6 @@ if(WIN32)
     add_compile_options(/source-charset:utf-8) # Force MSVC to compile source as UTF-8.
     add_compile_options(/Zi) # enable debugging information
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /DEBUG")
-
```

Comment:
Is it necessary to remove it here, now that there is `REMOVE_ITEM ` for it in the ObjWriter part?

Reply:
I could have sworn I tried that before and it didn't work, but tried again and it did...